appserver/monitoring-console/webapp/src/main/webapp/js/mc-view-units.js

```diff
@@ -115,6 +115,21 @@ MonitoringConsole.View.Units = (function() {
       percent: PERCENT_FACTORS,
    };
 
+   const UNIT_NAMES = {
+      count: 'Count', 
+      ms: 'Milliseconds', 
+      ns: 'Nanoseconds', 
+      bytes: 'Bytes', 
```

Comment:
Is bytes necessarily the best unit vs. kilo or mega?
An concrete example in our monitoring doesn't jump immediately to mind but just inciting the thought.

Reply:
Its more a dimension or base unit. KB or MB are multiples of this base unit. Which is expressed further up in the table of conversions. Also metrics themselves are actually given in bytes not kilo or mega. For the user this has no significance as values are automatically shown in the appropriate unit(s). For example 5KB or 10MB300Kb.  Actual metrics such as used memory are in bytes and are only converted to something more fitting in the UI.